integration/php_nginx_test.go

```diff
@@ -0,0 +1,81 @@
+package integration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testPhpNginx(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack()
+		docker = occam.NewDocker()
+	})
+
+	context("when building a php app using php-web, composer, and nginx", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			fmt.Printf("test image name %s\n", image.ID)
+			// Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
```

Comment:
```suggestion
			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
```
Is this commented out for a reason?